src/components/binding/binding-loader.js

```diff
@@ -258,6 +258,5 @@ genotet.BindingLoader.prototype.loadBindingList = function() {
     data.forEach(function(dataInfo) {
       genotet.data.bindingGenes.push(dataInfo.gene);
     });
-    this.signal('track');
```

Comment:
If you don't signal track here, the panel.updateTracks is not called. Are you sure this still runs correctly? Can user still change tracks or add tracks properly in the ui?